EMacco/atm-simulation
# #168778865 Setup react environment

#### What does this PR do?
This PR sets up the front end using react and redux

#### Description of Task to be completed?
- Setup react
- Setup redux
- Setup Tailwindcss
- Setup react router
- Create 404 page

#### How should this be manually tested?
- Checkout to this branch
- Run `rails s`
- Open your browser and navigate to `localhost:3000`
- You should see `This is the Home`

#### What are the relevant pivotal tracker stories?
[#168778865](https://www.pivotaltracker.com/story/show/168778865)
